webapp/components/PostCard/PostCard.vue

```diff
@@ -176,6 +176,8 @@ export default {
   }
 
   .content-menu {
+    position: relative;
+    z-index: $z-index-post-card-link;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:36:14Z" title="Tuesday, January 14th 2020, 6:36:14 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -176,6 +176,8 @@ export default {
   }
 
   .content-menu {
+    position: relative;
+    z-index: 99;
```


I can't seem to figure out exactly why this is needed... if I comment out `z-index`, I don't seem to see any difference. with `position: relative`, the content menu has a `hover` affect that changes it to green, which I like by the way, but I don't notice any difference with or without the `z-index`.
Do we need it? If so, could we use a token?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:47:30Z" title="Wednesday, January 15th 2020, 8:47:30 am +01:00">Jan 15, 2020</time>_

---


@mattwr18 you're starting to sound like me! 😜 Of course this should be a design token.
I added the `z-index` for the same reason we added it on the `user-details`: It needs to be on top of the `post-link` on the index page to be clickable.